### ISSUE TITLE
Command-line export of .wav, .mp3 and .flac files is broken in MuseScore 4.1.1 with Muse Sounds

### ISSUE THREAD

**Issue by KoenTanghe** [musescore/MuseScore#19160]
### Issue type

Other type of issue

### Bug description

It is possible to use the command-line to export various files from a score, using the -o option (midi, wav, mp3, png, ...). 
This was possible in MS3 and also in MS4, following this documentation of the MuseScore command line options: https://musescore.org/en/handbook/4/command-line-options

At some point, this broke in MS4 for the .mpos and .spos file types, but that was eventually fixed by @DmitryArefiev or @Eism  after I reported the problem in this issue a while ago: https://github.com/musescore/MuseScore/issues/16710

But now it turns out that export of **audio** files is broken in the current MuseScore 4 version.
You do get audio files, but they are now completely silent.

I am attaching a zip file with the following very simple test files and result files:

- test_MS3.mscz = simple test score created and saved with MS3
- test_MS4.mscz = same score opened and then saved with MS4
- test_MS3_exp3.mp3 = result of exporting test_MS3.mscz with MS3 command-line
- test_MS3_exp4.mp3 = result of exporting test_MS3.mscz with MS4 command-line
- test_MS4_exp4.mp3 = result of exporting test_MS4.mscz with MS3 command-line

[MS export bug.zip](https://github.com/musescore/MuseScore/files/12431324/MS.export.bug.zip)

### Steps to reproduce

You can use the following commands to reproduce this easily:

```
"C:\Program Files\MuseScore 3\bin\MuseScore3.exe" -o "test_MS3_exp3.mp3" test_MS3.mscz
"C:\Program Files\MuseScore 3\bin\MuseScore3.exe" -o "test_MS4_exp3.mp3" test_MS4.mscz
"C:\Program Files\MuseScore 4\bin\MuseScore4.exe" -o "test_MS3_exp4.mp3" test_MS3.mscz
"C:\Program Files\MuseScore 4\bin\MuseScore4.exe" -o "test_MS4_exp4.mp3" test_MS4.mscz
```

result command 1: OK, stereo sound file, properly rendered
result command 2: no result, kind of expected because MuseScore3 can't handle MuseScore 4 files
result command 3: **NOT OK: completely silent stereo sound file**
result command 4: **NOT OK: completely silent stereo sound file**

If it makes any difference, I have installed MuseScore 4 with Muse Sounds (default setup).

Edit: corrected initial report that MS3 exported sound file was mono: it's not, it's stereo.

### MuseScore Version

4.1.1

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Windows 10 Pro 64-bit (latest updates)

**Comment by cbjeukendrup** [comment]
I seem to recall this has come up before. The MuseSampler module is not initialised in command line mode. Trivial to fix, fortunately.

**Comment by KoenTanghe** [comment]
Quick follow-up, as I had some time to test this on another PC:

1) Small correction for what I said above about the MS3 export: the sound file is also stereo, not mono.

2) I just tried this on another PC, where I don't have Muse Hub / Muse Sounds installed, and there I do get sound rendered in the audio files. 
The sound is very quiet though: -37.2 dBFS for the MS4 export, versus -15.2 dBFS for the MS3 export (I expected some difference, but that is a lot).
I've attached the results on that second PC:

[MS export bug 2.zip](https://github.com/musescore/MuseScore/files/12433669/MS.export.bug.2.zip)

**Comment by DmitryArefiev** [comment]
Fixed in #19217

**Comment by KoenTanghe** [comment]
Thanks for the fix!
Will this be in the next MS4 release? And if so, when would that be?

Regarding the very low volume compared to a MuseScore 3 export: it looks like this is due to another regression bug, where in MS4 normalization during an audio export is not available any longer. Found an issue for that here: #16105  I will follow-up there on that.

**Comment by cbjeukendrup** [comment]
The fix will be included in 4.2, which is to be released somewhere in November.
